README.md

```diff
@@ -83,20 +92,32 @@ var app = express();
 ```
 
 ```js
-// Your answer...
+app.get("/", function(req, res){
+  res.render("index")
+});
+app.post("/", function(req, res){
+  res.send("create")
+});
+app.put("/:id", function(req, res){
+  res.send("update")
+});
+app.delete("/:id", function (req, res){
+  res.send("delete")
+});
+
 ```
 ### Question #7
 
 Describe the differences between Express and Rails as backend frameworks.
 
 ```text
-
+Rails is more structured and you have to do things a certain way. With express you have more freedom in how you setup the backend.
 ```
 
 ### Question #8
 
-What is the importance of using body-parser in our express application for post requests? 
+What is the importance of using body-parser in our express application for post requests?
 
 ```js
-
+Body-parser is what enables express to read forms and use user inputs.
 ```
```

Comment:
👍  Enables us to read the `body` of the `post` request fired after a user submits a form...